<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>video</class>
 <widget class="QDialog" name="video">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>47</y>
     <width>692</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="5">
        <widget class="QToolButton" name="stopBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/images/stop.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="speedBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/images/speed.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QSlider" name="soundSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QToolButton" name="soundBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/images/sound.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="videoSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="playPauseBtn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/images/play.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="10">
        <widget class="QToolButton" name="settingsBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/images/settins.png</normaloff>:/images/images/settins.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="speedLbl">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <widget class="QToolButton" name="playlistBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/images/list.png</normaloff>:/images/images/list.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QGroupBox" name="playlistBox">
      <property name="title">
       <string>Playlist</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="playListView"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="removeVideo">
           <property name="toolTip">
            <string>Remove selected item form playlist</string>
           </property>
           <property name="toolTipDuration">
            <number>1</number>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="clearBtn">
           <property name="toolTip">
            <string>Clear all items from the playlist</string>
           </property>
           <property name="toolTipDuration">
            <number>1</number>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="videoPlayerBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="actionPlaylist"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <addaction name="actionPlayAndPause"/>
    <addaction name="actionStop"/>
    <addaction name="actionSpeed"/>
    <addaction name="actionSound"/>
    <addaction name="separator"/>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_MediaPlayer"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPlayAndPause">
   <property name="text">
    <string>Play or Pause</string>
   </property>
   <property name="toolTip">
    <string>Play or Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSpeed">
   <property name="text">
    <string>Speed</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionSound">
   <property name="text">
    <string>Change volume</string>
   </property>
   <property name="toolTip">
    <string>Mute / Play sound</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionAbout_MediaPlayer">
   <property name="text">
    <string>About MediaPlayer</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPlaylist">
   <property name="text">
    <string>Open Playlist</string>
   </property>
   <property name="toolTip">
    <string>Open a playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New window</string>
   </property>
   <property name="toolTip">
    <string>Open new window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
